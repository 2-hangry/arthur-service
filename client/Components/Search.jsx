import React from 'react';

const Search = props => (
  <form onSubmit={props.submit}>
    <label>
        Restaurant:
      <input type="number" onChange={props.handleChange} />
    </label>
    <input type="submit" value="Submit" />
  </form>
);

export default Search;
