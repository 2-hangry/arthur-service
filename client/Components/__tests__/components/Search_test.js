import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import Search from '../../Search.jsx';

Enzyme.configure({ adapter: new Adapter() });

describe('restaurant Suite', function() {
  it('should render without crashing', () => {
    const wrapper = shallow(<Search />)
    expect(wrapper.exists()).toEqual(true);
  });
});