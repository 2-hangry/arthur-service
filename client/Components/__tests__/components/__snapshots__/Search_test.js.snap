// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<form>
  <label>
    Restaurant:
    <input
      type="number"
    />
  </label>
  <input
    type="submit"
    value="Submit"
  />
</form>
`;
